.github/pull_request_template.md

```diff
@@ -5,3 +5,17 @@
 @giantswarm/team-cabbage will be automatically requested for review once
 this PR has been submitted.
 -->
+
+This PR...
+
+## Checklist
+
+- [ ] Automated test are working
+- [ ] Changelog entry has been added
+
+### Manual tests on workload clusters (not always required)
```

Comment:
Can we change to something like, required on chart changes etc. i.e - when would they be required?

Reply:
yes